ding_periodical.module

```diff
@@ -135,12 +136,17 @@ function theme_ding_periodical_issues($variables) {
 
       // Check if the volume is reservable and add reservation button.
       if (!empty($issue_id) && ding_periodical_is_reservable($issues[$key])) {
-        $path = array($issue_id, $volume, $key);
+        $item_id = array($issue_id, $volume, $key);
+
+        foreach ($item_id as &$v) {
+          $v = rawurlencode($v);
```

Comment:
As I read the comments in the issue I see this as a problem with DDELibra/Alma - not with periodicals in general.

If that is the case then the encoding problem should be fixed [in the provider](https://github.com/ding2/alma/blob/master/includes/alma.reservation.inc#L228) - not in the general periodical handling module. Here the change will affect all providers potentially causing other problems.